.github/workflows/backport.yml

```
@@ -0,0 +1,241 @@
+              dry_run="true"
+              break
+            fi
+          done
+          
+          # Validate inputs
+          if [[ -z "$branch_name" ]]; then
+            echo "::error::Target branch is required."
+            echo "::error::Usage: /backport release-X.Y [--dry-run]"
+            exit 1
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)